config/secrets.yml

```diff
@@ -17,4 +17,4 @@ test:
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
```

Comment:
I would revert this change. It isn't germane to the PR, and the key should be indented.